src/pages/[slug]/[ie]/index.tsx

```diff
@@ -70,6 +72,9 @@ const ObjectDetailPage: NextPage = () => {
 	const { t } = useTranslation();
 	const router = useRouter();
 	const previousUrl = useSelector(selectPreviousUrl);
+	const showResearchWarning = useSelector((state: AppState) =>
+		selectHasPermission(state, Permission.SHOW_RESEARCH_WARNING)
+	);
```

Comment:
i wonder if we could wrap this into a hook? Since we'll be using something like this a lot in the client.
something like:
```
const showResearchWarning: boolean = useHasPermission(Permission.SHOW_RESEARCH_WARNING);
```

We can do that in a separate PR though

Reply:
Waarschijnlijk wel. Dan kunnen we ineens support toevoegen om tegen meerdere permissies af te checken.

Reply:
https://meemoo.atlassian.net/browse/ARC-778